packages/table/stories/table.stories.tsx

```diff
@@ -0,0 +1,505 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+import { Table, TableProps } from '../src';
+import { Icon } from '@sk-web-gui/icon';
+import { Button } from '@sk-web-gui/button';
+import { AutoTableHeader, TableHeader, AutoTableColumn, sortMode } from '../src/auto-table';
+import { Input, Select } from '@sk-web-gui/forms';
+import { Pagination } from '@sk-web-gui/pagination';
+import _ from 'lodash';
+
+export default {
+  title: 'Komponenter/Table/Table',
+  component: Table,
+  tags: ['autodocs'],
+} as Meta<typeof Table>;
+
+//eslint-disable-next-line
+type TableValue = any;
+type TableItem = Record<string | number, TableValue>;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const headerLabels = [{ label: 'Column 1' }, { label: 'Column 2' }, { label: 'Column 3' }];
+
+const data = [
+  {
+    id: 1,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+  {
+    id: 2,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+  {
+    id: 3,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+  {
+    id: 4,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+  {
+    id: 5,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+  {
+    id: 6,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+
+  {
+    id: 7,
+    column1: 'cell',
+    column2: 'cell',
+    column3: 'cell',
+  },
+];
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const rows: any = data.map((d, idx: number) => {
+  return (
+    <Table.Row key={`row-${idx}`}>
+      <Table.Column>
+        <span>{d.column1}</span>
+      </Table.Column>
+      <Table.Column>
+        <span>{d.column2}</span>
+      </Table.Column>
+      <Table.Column>
+        <span>{d.column3}</span>
+      </Table.Column>
+    </Table.Row>
+  );
+});
+
+export const Template = (args: TableProps) => (
+  <div className="flex flex-col gap-16">
+    <Table className="table-column-group border-none rounded-0" {...args}>
+      <Table.Header>
+        {headerLabels.map((l, idx) => {
+          return (
+            <Table.HeaderColumn key={`header-${idx}`}>
+              <span>{l.label}</span>
+            </Table.HeaderColumn>
+          );
+        })}
+      </Table.Header>
+      <tbody>
+        {rows.map((row) => {
+          return row;
+        })}
+      </tbody>
+    </Table>
+  </div>
+);
+
+Template.story = { name: 'Table' };
+
+export const DataTable = (args: TableProps) => {
```

Comment:
No need to make this a full AutoTable. 
Make this much more simple, but still as an example of how to use the Table as a DataTable.